core/templates/domain/recommendations/exploration-recommendations-backend-api.service.ts

```diff
@@ -25,11 +25,22 @@ import {
   LearnerExplorationSummaryBackendDict
 } from 'domain/summary/learner-exploration-summary.model';
 
+// The type below is used to create an object that is passed to a HTTP get
+// method. When using the interface style, TypeScript throws an error because
+// this type and the type of the method's expected argument don't match
+// exactly (even though they are compatible).
+// The solution is to either change this type definition so it exactly matches
+// the method's expected argument type, or use the type alias style instead of
+// interface (this requires us to explicitly allow the use of type alias).
+// The second solution has been used to preserve type safety.
+
```

Comment:
```suggestion
// Eslint disable is added because we need to use 'type' instead of 'interface'
// to create the 'RecommendationsUrlParams' type. This is done, because 
// when using the interface style, TypeScript throws an error because
// this type and the type of the method's expected argument don't match
// exactly (even though they are compatible).
```

Reply:
Done.